dms-core/src/main/kotlin/team/aliens/dms/domain/student/service/GetStudentServiceImpl.kt

```diff
@@ -0,0 +1,64 @@
+package team.aliens.dms.domain.student.service
+
+import team.aliens.dms.common.annotation.Service
+import team.aliens.dms.common.spi.SecurityPort
+import team.aliens.dms.domain.manager.dto.PointFilter
+import team.aliens.dms.domain.manager.dto.Sort
+import team.aliens.dms.domain.school.validateSameSchool
+import team.aliens.dms.domain.student.exception.StudentNotFoundException
+import team.aliens.dms.domain.student.model.Student
+import team.aliens.dms.domain.student.spi.QueryStudentPort
+import java.util.UUID
+
+@Service
+class GetStudentServiceImpl(
+    private val securityPort: SecurityPort,
+    private val queryStudentPort: QueryStudentPort
+) : GetStudentService {
+
+    override fun getCurrentStudent(): Student {
+        val currentUserId = securityPort.getCurrentUserId()
+        return queryStudentPort.queryStudentById(currentUserId) ?: throw StudentNotFoundException
```

Comment:
```suggestion
        return queryStudentPort.queryStudentByUserId(currentUserId) ?: throw StudentNotFoundException
```
여기 userId 외래키로 조회해야되는 거 아닌가요?

Reply:
헉 맞아요